baseplate/experiments/providers/multi_variant.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+from .simple_experiment import SimpleExperiment
+
+
+class MultiVariantExperiment(SimpleExperiment):
+    """Basic experiment, handling more than two variants (typically two
```

Comment:
summary line is supposed to be separated from the rest and generally fit on one line.

we need to get this into CI, but can you run the `pep257` tool on this please?

Reply:
This file has been removed.